README.md

```diff
@@ -0,0 +1,111 @@
+# Realtime Pollen Calibration
+
+This project translates the pollen calibration module from FORTRAN into Python.
+More information about the Pollen module currently in the weather model COSMO can be found here:
+<https://service.meteoswiss.ch/confluence/x/dYQYBQ>
+And in this paper: (currently under review :)
```

Comment:
Link to Preprint: https://assets.researchsquare.com/files/rs-2254663/v1/f7f26422c6b4615c2f7e1dfe.pdf?c=1668761631